.devops/templates/pr-ci.yml

```diff
@@ -0,0 +1,159 @@
+parameters:
+  - name: isPR
+    type: boolean
+
+variables:
+  - group: fabric-variables
+  - template: variables.yml
+  - ${{ if eq(parameters.isPR, true) }}:
+      - sinceArg: --since $(targetBranch)
+
+pool: 'Self Host Ubuntu'
+
+jobs:
+  - job: BuildTestLint
+    workspace:
+      clean: all
+    steps:
+      - template: tools.yml
+
+      - task: Bash@3
+        inputs:
+          inputs:
+          filePath: yarn-ci.sh
+        displayName: yarn
+
+      - script: |
+          yarn checkchange
+        displayName: check change
+
+      ## manually call this since lage is now used and it calls lint per package, not per repo
+      - script: |
+          yarn prelint &&
+          yarn prettier $(targetBranch) --check
+        displayName: do syncpack, lint-files, and prettier checks
+
+      ## Danger.js checks for Fluent UI N*
```

Comment:
with the recent work on separating v0 from v7/8 would it make sense to just spin up a separate pipeline for v0 ? 

We removed dependency on datetime and keyboard key, I don't think there's much left that really ties it with the rest of the repo

Reply:
Maybe, but the trouble is figuring out how to do the scoping. Also it's a bit of an orthogonal problem to what I'm trying to solve here.

Reply:
I agree, just wanted to float the idea, it will also make both pipelines (if we get there) a lot lot simpler